docs/reference/cli.rst

```diff
@@ -445,6 +445,10 @@ import ...``.
   specified, the plugin list in your configuration is ignored. The long form
   of this argument also allows specifying no plugins, effectively disabling
   all plugins: ``--plugins=``.
+* ``-P EXCLUDE``: specify a comma-separated list of plugins to disable in a
+  specific beets run. If specified, it will exclude plugins from your configuration
+  and/or plugins specified using the ``-p`` flag. To disable ALL plugins, use
```

Comment:
Not sure we need the second sentence? It seems like this is pretty logical.

Reply:
I agree there is some redundancy here, but I think it is worth mentioning that it overwrites `-p`?

Reply:
Ah, sure, that seems worth preserving.